seacrowd/sea_datasets/copal/copal.py

```diff
@@ -0,0 +1,157 @@
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@article{wibowo2023copal,
+  title={COPAL-ID: Indonesian Language Reasoning with Local Culture and Nuances},
+  author={Wibowo, Haryo Akbarianto and Fuadi, Erland Hilman and Nityasya, Made Nindyatama and Prasojo, Radityo Eko and Aji, Alham Fikri},
+  journal={arXiv preprint arXiv:2311.01012},
+  year={2023}
+}
+"""
+_DATASETNAME = "copal"
+
+_DESCRIPTION = """\
+COPAL is a novel Indonesian language common sense reasoning dataset. Unlike the previous Indonesian COPA dataset (XCOPA-ID), COPAL-ID incorporates Indonesian local and cultural nuances,
+providing a more natural portrayal of day-to-day causal reasoning within the Indonesian cultural sphere.
+Professionally written by natives from scratch, COPAL-ID is more fluent and free from awkward phrases, unlike the translated XCOPA-ID.
+Additionally, COPAL-ID is presented in both standard Indonesian and Jakartan Indonesian–a commonly used dialect.
+It consists of premise, choice1, choice2, question, and label, similar to the COPA dataset.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/haryoaw/COPAL"
+
+_LICENSE = Licenses.CC_BY_SA_4_0.value
+
+_URLS = {
+    _DATASETNAME: {"test": "https://huggingface.co/datasets/haryoaw/COPAL/resolve/main/test_copal.csv?download=true", "test_colloquial": "https://huggingface.co/datasets/haryoaw/COPAL/resolve/main/test_copal_colloquial.csv?download=true"},
+}
+
+_SUPPORTED_TASKS = [Tasks.QUESTION_ANSWERING]
```

Comment:
btw, we're gonna introduce `COMMONSENSE_REASONING` task under this [PR](https://github.com/SEACrowd/seacrowd-datahub/pull/85). please wait for that PR and re-implement once it has been approved and merged. The changes will be minimum tho, you just need to add `meta` column in SEACrowd Schema

Reply:
got it, thanks!

Reply:
Hi @williamnixon20, the PR has been merged recently. Please pull it to your branch/fork and re-implement it. I can help on guiding that if needed 😃

Reply:
Thanks for offer, appreciate it!  I already added the meta tag and changed the task. As you said, the changes were minimum 😄